os/manual-rollbacks.md

```diff
@@ -110,17 +110,25 @@ $ cgpt find -t coreos-usr
 /dev/sda4
 ```
 
-To figure out which partition is currently active, we can use `findmnt`.
+To figure out which partition is currently active, we can use `lsblk`.
 
 ```
-$ findmnt --noheadings --raw --output=source --target=/usr
-/dev/sda4
+$ lsblk --output NAME,TYPE,MOUNTPOINT
+NAME    TYPE  MOUNTPOINT
+sda     disk  
+...
+|-sda4  part  
+| `-usr crypt /usr
+...
 ```
 
+Note: Older Container Linux systems will not have an additional "crypt" partition.
```

Comment:
Can we explain why or what it is? Any maybe attempt to rephrase so it doesn't have to say `Note:`?